### PR TITLE
enhancement (#112) Fixed GIF display issue and improved design for Downtime molecule

### LINKED ISSUE #112: Enhancement: Fix GIF Display Issue and Improve Design for Downtime Molecule
image

### PR DESCRIPTION
**Pull request by ma1hotraz**
closes #112 

Mobile-
image

Desktop-
image

### CODE REVIEW

**Comment by prtkjakhar** [comment]
Hey @ma1hotraz gif not visible on deployed preview link

**Comment by ma1hotraz** [comment]
> Hey @ma1hotraz gif not visible on deployed preview link

checking

**Comment by ma1hotraz** [comment]
@prtkjakhar fixed, please check. [PR](https://ui-templates-9wz4uvx7x-samagra-uci.vercel.app/)

**Comment by prtkjakhar** [comment]
> @prtkjakhar fixed, please check. [PR](https://ui-templates-9wz4uvx7x-samagra-uci.vercel.app/)

you have hardcoded the downtimegif, need to take it from config only

**Comment by ma1hotraz** [comment]
> > @prtkjakhar fixed, please check. [PR](https://ui-templates-9wz4uvx7x-samagra-uci.vercel.app/)
> 
> you have hardcoded the downtimegif, need to take it from config only

@prtkjakhar If the image is in the 'src' folder, it can be used by importing it, as I have done. However, the issue with using images directly from the 'src' folder is that when the bundler tries to resolve their paths, it doesn't find them because they're not _served statically_. Placing images in the 'public' folder allows them to be served statically, so their paths work as expected. This enables you to reference them directly in your HTML or JSX without importing.

Since you wanted to use an image from the config file, I moved the GIF to the 'public' folder and referenced it directly in the downtime component. The images not appearing in the UI are also due to the same issue. I'll raise an issue and fix them in the next PR.

Please check to see if I have resolved this one correctly. And assign this to me.

**Comment by ma1hotraz** [comment]
@prtkjakhar

File: src/molecules/downtime-page/index.tsx [hunk 1]
```diff
@@ -1,115 +1,96 @@
...
+  const config = useUiConfig("component", "downtime");
+  const theme = useColorPalates();
 
-  const theme = useColorPalates()
   const handleRefreshClick = useCallback(() => {
-    // window?.location.reload()
-    console.log(config.refreshText ?? 'Contact Details')
-  }, [])
+    console.log(config.refreshText ?? "Reload Page");
+  }, [config.refreshText]);
```
Comment (prtkjakhar) [inline src/molecules/downtime-page/index.tsx]: remove these dependencies

File: src/molecules/downtime-page/index.tsx [hunk 2]
```diff
@@ -1,115 +1,96 @@
...
+
   const handlePreviousClick = useCallback(() => {
-    // window?.history.back();
-    console.log(config.previousPageText ?? 'Contact Details')
-  }, [])
+    console.log(config.previousPageText ?? "Previous Page");
+  }, [config.previousPageText]);
```
Comment (prtkjakhar) [inline src/molecules/downtime-page/index.tsx]: this also

File: src/molecules/downtime-page/index.tsx [hunk 3]
```diff
@@ -1,115 +1,96 @@
...
 
   const handleContactUserClick = useCallback(() => {
-    console.log(config.contactLink ?? 'Contact Details')
-  }, [])
+    console.log(config.contactLink ?? "Contact Details");
+  }, [config.contactLink]);
```
Comment (prtkjakhar) [inline src/molecules/downtime-page/index.tsx]: this also

**Review by prtkjakhar — CHANGES_REQUESTED** [review]

File: src/molecules/downtime-page/index.tsx (hunk 1 above)
Comment (ma1hotraz) [reply src/molecules/downtime-page/index.tsx]: done

File: src/molecules/downtime-page/index.tsx (hunk 2 above)
Comment (ma1hotraz) [reply src/molecules/downtime-page/index.tsx]: done

File: src/molecules/downtime-page/index.tsx (hunk 3 above)
Comment (ma1hotraz) [reply src/molecules/downtime-page/index.tsx]: done

**Comment by ma1hotraz** [comment]
@prtkjakhar removed those dependencies. Please check

**Pull request merged**